guides/creating-a-custom-stepper-using-the-cdk-stepper.md

```diff
@@ -0,0 +1,153 @@
+# Creating a custom stepper using the CDK stepper
+
+Using the [CDK stepper](https://material.angular.io/cdk/stepper/overview) it is possible to build a custom stepper which you can completely style yourself without any specific Material Design styling.
+
+In this guide, we'll learn how we can build our own custom stepper using the CDK stepper.
+
+## Add CDK stepper to your project 
+
+Add the CDK to your Angular project using npm or yarn:
```

Comment:
I'm not sure whether we should be repeating the setup guide since the one here will probably get out of sync.

Reply:
Good point, removed it